Translation is broken on Paper 1.21.1

### Server Logs
```
[21:53:02 INFO]: [AntiRedstoneClock-Remastered] Loading server plugin AntiRedstoneClock-Remastered v2.7.1
[21:53:02 INFO]: [net.onelitefeather.antiredstoneclockremastered.service.factory.RegionServiceFactory] Using BukkitRegionService
[21:53:02 INFO]: [net.onelitefeather.antiredstoneclockremastered.service.factory.SchedulerServiceFactory] Using BukkitSchedulerService
[21:53:02 INFO]: [net.onelitefeather.antiredstoneclockremastered.injection.TranslationModule] Using modern translation service
[21:53:02 WARN]: [net.onelitefeather.antiredstoneclockremastered.injection.ExternalSupportModule] PlotSquared hasn't been found!
[21:53:02 WARN]: [net.onelitefeather.antiredstoneclockremastered.injection.ExternalSupportModule] WorldGuard hasn't been found!
[21:53:03 INFO]: [net.onelitefeather.antiredstoneclockremastered.service.factory.RedstoneClockServiceFactory] Using BukkitRedstoneClockService
[21:53:03 ERROR]: [AntiRedstoneClock-Remastered] Error initializing plugin 'AntiRedstoneClock-Remastered-2.7.1.jar' in folder 'plugins' (Is it up to date?)
com.google.inject.CreationException: Unable to create injector, see the following errors:
1) [Guice/ErrorInCustomProvider]: NoClassDefFoundError: net/kyori/adventure/text/minimessage/translation/MiniMessageTranslationStore
  at TranslationModule.provideTranslationService(TranslationModule.java:37)
  while locating TranslationService
Learn more:
  https://github.com/google/guice/wiki/ERROR_IN_CUSTOM_PROVIDER
1 error
======================
Full classname legend:
======================
TranslationModule:    "net.onelitefeather.antiredstoneclockremastered.injection.TranslationModule"
TranslationService:   "net.onelitefeather.antiredstoneclockremastered.service.api.TranslationService"
========================
End of classname legend:
========================
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.Errors.throwCreationExceptionIfErrorsExist(Errors.java:589) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:190) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:113) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.Guice.createInjector(Guice.java:87) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.AntiRedstoneClockRemastered.onLoad(AntiRedstoneClockRemastered.java:36) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at io.papermc.paper.plugin.storage.ServerPluginProviderStorage.processProvided(ServerPluginProviderStorage.java:59) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at io.papermc.paper.plugin.storage.ServerPluginProviderStorage.processProvided(ServerPluginProviderStorage.java:18) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at io.papermc.paper.plugin.storage.SimpleProviderStorage.enter(SimpleProviderStorage.java:39) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at io.papermc.paper.plugin.entrypoint.LaunchEntryPointHandler.enter(LaunchEntryPointHandler.java:40) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at org.bukkit.craftbukkit.CraftServer.loadPlugins(CraftServer.java:547) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:292) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1214) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:329) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.NoClassDefFoundError: net/kyori/adventure/text/minimessage/translation/MiniMessageTranslationStore
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.service.impl.ModernTranslationService.<init>(ModernTranslationService.java:17) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.injection.TranslationModule.provideTranslationService(TranslationModule.java:43) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.injection.TranslationModule$$FastClassByGuice$$18376f.GUICE$TRAMPOLINE(<generated>) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.injection.TranslationModule$$FastClassByGuice$$18376f.apply(<generated>) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.ProviderMethod$FastClassProviderMethod.doProvision(ProviderMethod.java:260) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.ProviderMethod.doProvision(ProviderMethod.java:171) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalProviderInstanceBindingImpl$CyclicFactory.provision(InternalProviderInstanceBindingImpl.java:185) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalProviderInstanceBindingImpl$CyclicFactory.get(InternalProviderInstanceBindingImpl.java:162) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:169) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:45) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalInjectorCreator.loadEagerSingletons(InternalInjectorCreator.java:213) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:186) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        ... 12 more
Caused by: java.lang.ClassNotFoundException: net.kyori.adventure.text.minimessage.translation.MiniMessageTranslationStore
        at io.papermc.paper.plugin.entrypoint.classloader.PaperPluginClassLoader.loadClass(PaperPluginClassLoader.java:146) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at io.papermc.paper.plugin.entrypoint.classloader.PaperPluginClassLoader.loadClass(PaperPluginClassLoader.java:107) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.service.impl.ModernTranslationService.<init>(ModernTranslationService.java:17) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.injection.TranslationModule.provideTranslationService(TranslationModule.java:43) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.injection.TranslationModule$$FastClassByGuice$$18376f.GUICE$TRAMPOLINE(<generated>) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.injection.TranslationModule$$FastClassByGuice$$18376f.apply(<generated>) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.ProviderMethod$FastClassProviderMethod.doProvision(ProviderMethod.java:260) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.ProviderMethod.doProvision(ProviderMethod.java:171) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalProviderInstanceBindingImpl$CyclicFactory.provision(InternalProviderInstanceBindingImpl.java:185) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalProviderInstanceBindingImpl$CyclicFactory.get(InternalProviderInstanceBindingImpl.java:162) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:169) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:45) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalInjectorCreator.loadEagerSingletons(InternalInjectorCreator.java:213) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at AntiRedstoneClock-Remastered-2.7.1.jar/com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:186) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        ... 12 more











[21:53:06 INFO]: [AntiRedstoneClock-Remastered] Enabling AntiRedstoneClock-Remastered v2.7.1
[21:53:06 ERROR]: Error occurred while enabling AntiRedstoneClock-Remastered v2.7.1 (Is it up to date?)
java.lang.NullPointerException: Cannot invoke "com.google.inject.Injector.getInstance(java.lang.Class)" because "this.injector" is null
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.AntiRedstoneClockRemastered.onEnable(AntiRedstoneClockRemastered.java:49) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:288) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:641) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:590) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:753) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:515) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:329) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1214) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:329) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
[21:53:06 INFO]: [AntiRedstoneClock-Remastered] Disabling AntiRedstoneClock-Remastered v2.7.1
[21:53:06 ERROR]: Error occurred while disabling AntiRedstoneClock-Remastered v2.7.1
java.lang.NullPointerException: Cannot invoke "com.google.inject.Injector.getInstance(java.lang.Class)" because "this.injector" is null
        at AntiRedstoneClock-Remastered-2.7.1.jar/net.onelitefeather.antiredstoneclockremastered.AntiRedstoneClockRemastered.onDisable(AntiRedstoneClockRemastered.java:62) ~[AntiRedstoneClock-Remastered-2.7.1.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:291) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.disablePlugin(PaperPluginInstanceManager.java:237) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.disablePlugin(PaperPluginManagerImpl.java:114) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:550) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:206) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:641) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:590) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:753) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:515) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:329) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1214) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:329) ~[paper-1.21.1.jar:1.21.1-133-3cb8529]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```
### Expected behavior

The plugin should load. 

### Actual behavior

The plugin did not load, throwing Adventure API-related errors.

### Steps to reproduce

Create a server using Paper 1.21.1, and put the plugin in the plugins folder.

### Plugin and Datapack List
```
[22:16:22 INFO]: Server Plugins (6):
[22:16:22 INFO]: Paper Plugins:
[22:16:22 INFO]:  - AntiRedstoneClock-Remastered
[22:16:22 INFO]: Bukkit Plugins:
[22:16:22 INFO]:  - Essentials, EssentialsChat, EssentialsSpawn, LuckPerms, Multiverse-Core
```
### Paper version

```
[22:16:40 INFO]: This server is running Paper version 1.21.1-133-ver/1.21.1@3cb8529 (2025-03-28T16:14:36Z) (Implementing API version 1.21.1-R0.1-SNAPSHOT)
You are running the latest version
Previous version: 1.21.1-2329-803bf62 (MC: 1.21.1)
```
### Plugin version
```
[22:07:18 INFO]: AntiRedstoneClock-Remastered version 2.7.1
[22:07:18 INFO]: Authors: OneLiteFeather and TheMeinerLP
```
### Other

_No response_

--- a/src/main/java/net/onelitefeather/antiredstoneclockremastered/injection/TranslationModule.java
+++ b/src/main/java/net/onelitefeather/antiredstoneclockremastered/injection/TranslationModule.java
@@ -33,9 +33,9 @@ public final class TranslationModule extends AbstractModule {
     protected void configure() {
     }
 
+    @SuppressWarnings("removal")
     @Provides
     @Singleton
-    @SuppressWarnings("deprecation")
     public TranslationService provideTranslationService() {
         if (isLegacyVersion()) {
             LOGGER.info("Using legacy translation service");

